sdk/python/tests/integration/registration/test_universal_odfv_feature_inference.py

```diff
@@ -27,14 +31,26 @@ def test_infer_odfv_features(environment, universal_data_sources, infer_features
         infer_features=infer_features,
     )
 
-    feast_objects = [driver_hourly_stats, driver_odfv, driver(), customer()]
+    sim_odfv = similarity_feature_view(
```

Comment:
is there a way to make a separate test that specifically tests the similarity odfv? so we still get redshift coverage for other functionality?

Reply:
same with other tests

Reply:
@adchia Good point. Let me isolate it...

Reply:
@achals @judahrand The unit test has been simplified. Now, the Java integration test is failing, and I'm not sure why. I haven't touched the Java code.